### PR TITLE
cove360: templates: base.html: logo link.

### PR DESCRIPTION
**Pull request by BibianaC**
Link logo image to 360 main webpage.
Relates to https://github.com/OpenDataServices/cove/issues/779

### CODE REVIEW

**Comment by BibianaC** [comment]
[Logo to link to homepage](https://trello.com/c/LfVjKcsr/115-logo-to-link-to-homepage)

File: cove_360/templates/cove_360/base.html [hunk 1]
```diff
@@ -8,7 +8,9 @@
 {% block heading %}
   <div class="row">
     <div class="col-md-5">
-      <a href="{% url 'index' %}"><img src="{% static 'dataexplore/360-giving-logo.svg' %}" alt="360Giving Logo" width="270"/></a>
+      <a id="360_giving_logo" href="http://www.threesixtygiving.org/">
```
Comment (rhiaro) [inline cove_360/templates/cove_360/base.html:11]:
```suggestion
      <a id="360_giving_logo" href="https://www.threesixtygiving.org/">
```

File: cove_360/tests_functional.py [hunk 2]
```diff
@@ -362,6 +362,14 @@ def test_index_page_360_links(server_url, browser, link_text, url):
     assert url in href
 
 
+def test_index_page_360_logo_link(server_url, browser):
+    browser.get(server_url)
+    link = browser.find_element_by_id('360_giving_logo')
+    href = link.get_attribute('href')
+
+    assert href == 'http://www.threesixtygiving.org/'
```
Comment (rhiaro) [inline cove_360/tests_functional.py:370]:
```suggestion
    assert href == 'https://www.threesixtygiving.org/'
```

**Review by rhiaro — CHANGES_REQUESTED** [review]

File: cove_360/templates/cove_360/base.html (hunk 1 above)
Comment (BibianaC) [reply cove_360/templates/cove_360/base.html]: Thanks @rhiaro for the suggestion.

File: cove_360/tests_functional.py (hunk 2 above)
Comment (BibianaC) [reply cove_360/tests_functional.py]: Thanks @rhiaro for the suggestion.

**Review by rhiaro — APPROVED** [review]

**Pull request merged**